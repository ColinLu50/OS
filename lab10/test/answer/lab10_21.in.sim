23099
582
793
454
793
453
