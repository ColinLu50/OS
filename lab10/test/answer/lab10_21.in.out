FCFS
336 -> 59 -> 336 -> 382 -> 361 -> 67 -> 271 -> 334 -> 200 -> 260 -> 153 -> 218 -> 384 -> 154 -> 82 -> 90 -> 0 -> 54 -> 71 -> 162 -> 289 -> 8 -> 315 -> 350 -> 297 -> 169 -> 241 -> 153 -> 122 -> 8 -> 423 -> 186 -> 179 -> 26 -> 221 -> 120 -> 379 -> 254 -> 148 -> 186 -> 438 -> 308 -> 264 -> 89 -> 292 -> 71 -> 33 -> 260 -> 443 -> 180 -> 336 -> 146 -> 299 -> 148 -> 49 -> 20 -> 227 -> 227 -> 346 -> 66 -> 266 -> 204 -> 290 -> 0 -> 170 -> 191 -> 7 -> 92 -> 323 -> 437 -> 367 -> 279 -> 120 -> 396 -> 416 -> 21 -> 187 -> 227 -> 27 -> 36 -> 358 -> 234 -> 77 -> 141 -> 96 -> 112 -> 335 -> 228 -> 279 -> 442 -> 301 -> 361 -> 457 -> 271 -> 89 -> 1 -> 79 -> 180 -> 310 -> 1 -> 227 -> 345 -> 182 -> 67 -> 23 -> 436 -> 359 -> 75 -> 28 -> 340 -> 211 -> 122 -> 250 -> 3 -> 117 -> 150 -> 340 -> 53 -> 381 -> 251 -> 124 -> 270 -> 176 -> 159 -> 444 -> 111 -> 167 -> 233 -> 419 -> 254 -> 378 -> 88 -> 228 -> 310 -> 29 -> 69 -> 314 -> 72 -> 14 -> 341 -> 56 -> 45 -> 154 -> 346 -> 352 -> 455 -> 218 -> 155 -> 401 -> 349 -> 186 -> 157 -> 252 -> 116 -> 239 -> 119 -> 383 -> 281 -> 114 -> 73
23099
SSTF
336 -> 336 -> 336 -> 335 -> 334 -> 340 -> 340 -> 341 -> 345 -> 346 -> 346 -> 349 -> 350 -> 352 -> 358 -> 359 -> 361 -> 361 -> 367 -> 378 -> 379 -> 381 -> 382 -> 383 -> 384 -> 396 -> 401 -> 416 -> 419 -> 423 -> 436 -> 437 -> 438 -> 442 -> 443 -> 444 -> 455 -> 457 -> 323 -> 315 -> 314 -> 310 -> 310 -> 308 -> 301 -> 299 -> 297 -> 292 -> 290 -> 289 -> 281 -> 279 -> 279 -> 271 -> 271 -> 270 -> 266 -> 264 -> 260 -> 260 -> 254 -> 254 -> 252 -> 251 -> 250 -> 241 -> 239 -> 234 -> 233 -> 228 -> 228 -> 227 -> 227 -> 227 -> 227 -> 221 -> 218 -> 218 -> 211 -> 204 -> 200 -> 191 -> 187 -> 186 -> 186 -> 186 -> 182 -> 180 -> 180 -> 179 -> 176 -> 170 -> 169 -> 167 -> 162 -> 159 -> 157 -> 155 -> 154 -> 154 -> 153 -> 153 -> 150 -> 148 -> 148 -> 146 -> 141 -> 124 -> 122 -> 122 -> 120 -> 120 -> 119 -> 117 -> 116 -> 114 -> 112 -> 111 -> 96 -> 92 -> 90 -> 89 -> 89 -> 88 -> 82 -> 79 -> 77 -> 75 -> 73 -> 72 -> 71 -> 71 -> 69 -> 67 -> 67 -> 66 -> 59 -> 56 -> 54 -> 53 -> 49 -> 45 -> 36 -> 33 -> 29 -> 28 -> 27 -> 26 -> 23 -> 21 -> 20 -> 14 -> 8 -> 8 -> 7 -> 3 -> 1 -> 1 -> 0 -> 0
582
SCAN
336 -> 335 -> 334 -> 323 -> 315 -> 314 -> 310 -> 308 -> 301 -> 299 -> 297 -> 292 -> 290 -> 289 -> 281 -> 279 -> 271 -> 270 -> 266 -> 264 -> 260 -> 254 -> 252 -> 251 -> 250 -> 241 -> 239 -> 234 -> 233 -> 228 -> 227 -> 221 -> 218 -> 211 -> 204 -> 200 -> 191 -> 187 -> 186 -> 182 -> 180 -> 179 -> 176 -> 170 -> 169 -> 167 -> 162 -> 159 -> 157 -> 155 -> 154 -> 153 -> 150 -> 148 -> 146 -> 141 -> 124 -> 122 -> 120 -> 119 -> 117 -> 116 -> 114 -> 112 -> 111 -> 96 -> 92 -> 90 -> 89 -> 88 -> 82 -> 79 -> 77 -> 75 -> 73 -> 72 -> 71 -> 69 -> 67 -> 66 -> 59 -> 56 -> 54 -> 53 -> 49 -> 45 -> 36 -> 33 -> 29 -> 28 -> 27 -> 26 -> 23 -> 21 -> 20 -> 14 -> 8 -> 7 -> 3 -> 1 -> 0 -> 0 -> 340 -> 341 -> 345 -> 346 -> 349 -> 350 -> 352 -> 358 -> 359 -> 361 -> 367 -> 378 -> 379 -> 381 -> 382 -> 383 -> 384 -> 396 -> 401 -> 416 -> 419 -> 423 -> 436 -> 437 -> 438 -> 442 -> 443 -> 444 -> 455 -> 457
793
C-SCAN
336 -> 335 -> 334 -> 323 -> 315 -> 314 -> 310 -> 308 -> 301 -> 299 -> 297 -> 292 -> 290 -> 289 -> 281 -> 279 -> 271 -> 270 -> 266 -> 264 -> 260 -> 254 -> 252 -> 251 -> 250 -> 241 -> 239 -> 234 -> 233 -> 228 -> 227 -> 221 -> 218 -> 211 -> 204 -> 200 -> 191 -> 187 -> 186 -> 182 -> 180 -> 179 -> 176 -> 170 -> 169 -> 167 -> 162 -> 159 -> 157 -> 155 -> 154 -> 153 -> 150 -> 148 -> 146 -> 141 -> 124 -> 122 -> 120 -> 119 -> 117 -> 116 -> 114 -> 112 -> 111 -> 96 -> 92 -> 90 -> 89 -> 88 -> 82 -> 79 -> 77 -> 75 -> 73 -> 72 -> 71 -> 69 -> 67 -> 66 -> 59 -> 56 -> 54 -> 53 -> 49 -> 45 -> 36 -> 33 -> 29 -> 28 -> 27 -> 26 -> 23 -> 21 -> 20 -> 14 -> 8 -> 7 -> 3 -> 1 -> 0 -> 0 -> 458 -> 457 -> 455 -> 444 -> 443 -> 442 -> 438 -> 437 -> 436 -> 423 -> 419 -> 416 -> 401 -> 396 -> 384 -> 383 -> 382 -> 381 -> 379 -> 378 -> 367 -> 361 -> 359 -> 358 -> 352 -> 350 -> 349 -> 346 -> 345 -> 341 -> 340
454
LOOK
336 -> 335 -> 334 -> 323 -> 315 -> 314 -> 310 -> 308 -> 301 -> 299 -> 297 -> 292 -> 290 -> 289 -> 281 -> 279 -> 271 -> 270 -> 266 -> 264 -> 260 -> 254 -> 252 -> 251 -> 250 -> 241 -> 239 -> 234 -> 233 -> 228 -> 227 -> 221 -> 218 -> 211 -> 204 -> 200 -> 191 -> 187 -> 186 -> 182 -> 180 -> 179 -> 176 -> 170 -> 169 -> 167 -> 162 -> 159 -> 157 -> 155 -> 154 -> 153 -> 150 -> 148 -> 146 -> 141 -> 124 -> 122 -> 120 -> 119 -> 117 -> 116 -> 114 -> 112 -> 111 -> 96 -> 92 -> 90 -> 89 -> 88 -> 82 -> 79 -> 77 -> 75 -> 73 -> 72 -> 71 -> 69 -> 67 -> 66 -> 59 -> 56 -> 54 -> 53 -> 49 -> 45 -> 36 -> 33 -> 29 -> 28 -> 27 -> 26 -> 23 -> 21 -> 20 -> 14 -> 8 -> 7 -> 3 -> 1 -> 0 -> 340 -> 341 -> 345 -> 346 -> 349 -> 350 -> 352 -> 358 -> 359 -> 361 -> 367 -> 378 -> 379 -> 381 -> 382 -> 383 -> 384 -> 396 -> 401 -> 416 -> 419 -> 423 -> 436 -> 437 -> 438 -> 442 -> 443 -> 444 -> 455 -> 457
793
C-LOOK
336 -> 335 -> 334 -> 323 -> 315 -> 314 -> 310 -> 308 -> 301 -> 299 -> 297 -> 292 -> 290 -> 289 -> 281 -> 279 -> 271 -> 270 -> 266 -> 264 -> 260 -> 254 -> 252 -> 251 -> 250 -> 241 -> 239 -> 234 -> 233 -> 228 -> 227 -> 221 -> 218 -> 211 -> 204 -> 200 -> 191 -> 187 -> 186 -> 182 -> 180 -> 179 -> 176 -> 170 -> 169 -> 167 -> 162 -> 159 -> 157 -> 155 -> 154 -> 153 -> 150 -> 148 -> 146 -> 141 -> 124 -> 122 -> 120 -> 119 -> 117 -> 116 -> 114 -> 112 -> 111 -> 96 -> 92 -> 90 -> 89 -> 88 -> 82 -> 79 -> 77 -> 75 -> 73 -> 72 -> 71 -> 69 -> 67 -> 66 -> 59 -> 56 -> 54 -> 53 -> 49 -> 45 -> 36 -> 33 -> 29 -> 28 -> 27 -> 26 -> 23 -> 21 -> 20 -> 14 -> 8 -> 7 -> 3 -> 1 -> 0 -> 457 -> 455 -> 444 -> 443 -> 442 -> 438 -> 437 -> 436 -> 423 -> 419 -> 416 -> 401 -> 396 -> 384 -> 383 -> 382 -> 381 -> 379 -> 378 -> 367 -> 361 -> 359 -> 358 -> 352 -> 350 -> 349 -> 346 -> 345 -> 341 -> 340
453
